perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -4795,6 +4802,19 @@ private void processNotificationsAfterCreation(PerunSession session, Application
 		}
 	}
 
+	/**
+	 * Checks if form item belongs to form with given id
+	 *
+	 * @param sess
+	 * @param formId id of application form
+	 * @param itemId id of form item
+	 * @return true if item belongs to the form, false otherwise
+	 */
+	private boolean checkItemBelongsToForm(PerunSession sess, int formId, int itemId) {
+		int storedFormId = jdbc.queryForInt("select form_id from application_form_items where id=?", itemId);
```

Comment:
Just a suggestion: runtime exception `IncorrectResultSizeDataAccessException` is possibly thrown by this query if the item is not in DB - maybe we could catch it and return false in such a case.

Reply:
good point, added